main.go

```diff
@@ -71,9 +71,8 @@ func startFileWatch(root string) {
 
 func startWebServer(r http.Handler) {
 	addr := flags.WebServerAddress()
-	port := *flags.WebServerPort
 
-	log.Printf("Welcome to DemoIt. Please, open http://localhost:%d", port)
+	log.Printf("Welcome to DemoIt. Please, open %s", addr)
```

Comment:
I can't click on the address anymore because it doesn't contain `http://` prefix

Reply:
Fixed